### DISCUSSION TITLE
HttpSensor response_check logs not showing for all the attempts

### DISCUSSION THREAD

**Discussion by a-alabdulslam** [apache/airflow#32550] [Q&A] [answered]
We just upgrade from 2.5.3 to 2.6.2 and now the logs for response_check function showing logs for the last attempt (when the check is valid)
anyone has an idea what happened?
HttpSensor mode="reschedule"

**Comment by potiuk** [comment, marked as answer]
Look at the changelog of the provider https://airflow.apache.org/docs/apache-airflow-providers-http/stable/changelog.html

**Reply by a-alabdulslam** [reply]
Ok from the changelog the severity changed from info to debug that's why, right?

**Reply by potiuk** [reply]
looks like

**Reply by potiuk** [reply]
You can always downgrade the provider if you need to have it BTW.

**Reply by a-alabdulslam** [reply]
will try that thank you
